#include <iostream>
#include <exception>
// untuk objek exception yang akan digunakan
#include <array>
// untuk objek array yang akan digunakan
using namespace std;

int main() {
	cout << "Awal Program" << endl; //Penanda 1:...
	try
	{

	}
	catch (const std::exception&)
	{

	}
}